Separated shapes' definitions from target declarations:

MFSY opened the pull request:
- Created provsh module where prov related shapes are stored as librairies
 - Associated in prov module prov vocabulary types with relevant shapes defined in provsh through target declarations
 - Replaced 'this' prefix mapping with @base keyword
 - Renamed QualifiedAssociationShape to AssociationShape
 - Renamed QualifiedUsageShape to UsageShape
 - Used nexus/core/schema/v0.2.0

Fixes #38 #40 #41 #42

annakristinkaufmann commented on modules/prov/src/main/resources/schemas/nexus/prov/organization/v0.1.0.json:
```diff
@@ -1,14 +1,14 @@
     }
   ],
   "@type": "nxv:Schema",
   "shapes": [
```
should there also be a provsh import and node?

annakristinkaufmann commented on modules/prov/src/test/resources/data/nexus/prov/activity/v0.1.0/ordering-activity1-PASS-c30-ignored.json:
```diff
@@ -0,0 +1 @@
+[{"@id":"http://example.com/Graduation"},{"@id":"http://example.com/doctor_smith","@type":["http://www.w3.org/ns/prov#Entity"]},{"@id":"http://example.com/graduation","@type":["http://www.w3.org/ns/prov#Activity","http://example.com/Graduation"],"http://www.w3.org/ns/prov#startedAtTime":[{"@value":"2012-04-15T13:00:00-04:00","@type":"http://www.w3.org/2001/XMLSchema#dateTime"}],"http://www.w3.org ...
```
Did you want to include this file in the PR?

MFSY replied:
No. Removed

annakristinkaufmann commented on modules/prov/src/main/resources/schemas/nexus/prov/organization/v0.1.0.json:
```diff
@@ -1,14 +1,14 @@
     }
   ],
   "@type": "nxv:Schema",
   "shapes": [
```
should there be an import of the respective provsh schema?

MFSY replied:
In W3C PROV-0, Organization is not strongly constrained so that applications can decide how to describe an Organization.

annakristinkaufmann replied:
I see! Thanks!

annakristinkaufmann commented on modules/prov/src/main/resources/schemas/nexus/prov/person/v0.1.0.json:
```diff
@@ -1,14 +1,14 @@
     }
   ],
   "@type": "nxv:Schema",
   "shapes": [
```
should there be an import of the respective provsh schema?

MFSY replied:
See comment related to Organization.

annakristinkaufmann commented on modules/prov/src/main/resources/schemas/nexus/prov/softwareagent/v0.1.0.json:
```diff
@@ -1,14 +1,14 @@
     }
   ],
   "@type": "nxv:Schema",
   "shapes": [
```
should there be an import of the respective provsh schema?

MFSY replied:
See comment related to Organization.

MFSY committed:
Separated shape definition with target declarations:

 - Created provsh module where prov related shapes are stored as librairies
 - Associated in prov module prov vocabulary types with relevant shapes defined in provsh through target declarations
 - Replaced 'this' prefix mapping with @base keyword
 - Renamed QualifiedAssociationShape to AssociationShape
 - Renamed QualifiedUsageShape to UsageShape
 - Used nexus/core/schema/v0.2.0

Fixes #38 #40 #41 #42

MFSY committed:
Bumped nexus prov schemas version to v1.0.0

MFSY committed:
Removed ordering-activity1-PASS-c30-ignored test
